### PR TITLE
vllm add support for PVC ReadMany

### PR DESCRIPTION
**Pull request by samos123**

### CODE REVIEW

File: charts/vllm/examples/readmanypvc.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This example requires first running
```
Comment (nstogner) [inline charts/vllm/examples/readmanypvc.yaml]: Maybe rename this with a `-values.yaml` suffix

File: charts/vllm/values.yaml [hunk 2]
```diff
@@ -2,6 +2,16 @@ replicaCount: 1
 # Change this if you want to serve another model
 model: mistralai/Mistral-7B-Instruct-v0.1
 
+readManyPVC:
```
Comment (nstogner) [inline charts/vllm/values.yaml:6]: Maybe add a comment about the known-cloud support for this... I think it is only GKE right now.

**Review by nstogner — APPROVED** [review]

File: charts/vllm/values.yaml (hunk 2 above)
Comment (samos123) [reply charts/vllm/values.yaml:6]: fixed, added it's GKE only for now

File: charts/vllm/examples/readmanypvc.yaml (hunk 1 above)
Comment (samos123) [reply charts/vllm/examples/readmanypvc.yaml]: fixed

**Pull request merged**